garrysmod/lua/includes/modules/player_manager.lua

```diff
@@ -296,6 +296,12 @@ AddValidHands( "dod_american", "models/weapons/c_arms_dod.mdl", 1, "10000000" )
 
 local Type = {}
 
+function GetTable()
+
+	return table.Copy( Type )

```

Comment:
```
] lua_run PrintTable(player_manager.GetTable())
> PrintTable(player_manager.GetTable())...

[ERROR] lua/includes/modules/player_manager.lua:301: attempt to index global 'table' (a nil value)
  1. GetTable - lua/includes/modules/player_manager.lua:301
   2. unknown - lua_run:1
```

Also since `player_manager` library doesn't only handle player classes, the function name should point to the fact that it's the player classes table, not player models, etc., so something like ```player_manager.GetPlayerClasses()``` or something.

Reply:
Also also is everyone happy with the callback being called `Destroy`?

Reply:
For the consistency I would propose "OnRemove" "OnPlayerClassChanged" "OnChange" or something similar

Reply:
However, we have Destroy methods in some classes in Gmod.
But this case this is more like DFrame "OnClose" rather than physics object destroying

Reply:
Thing is nothing here is actually removed, and none of the other PLAYER: callbacks use `On` in their names. Something like `PLAYER:SwitchedAway` or `PLAYER:ChangedFrom` would better represent what is actually happening when the callback gets called.

Reply:
ClassChanged